### PR TITLE
Fix sle15sp6 pip install failed

### PR DESCRIPTION
**Pull request by lilyeyes**
Fix sle15sp6 pip install failed.
The original failure is `Test died: command 'pip install $(ls /hana/shared/HA1/hdbclient/hdbcli-*.tar.gz)' failed`.

Also fixed some tidy errors.
The reason why add `"--components=server,client"` to line 269:
CI check of `Running compile tests ` reports error `Possible attempt to separate words with commas at tests/sles4sap/hana_install.pm line 259.`

TEAM-8787 - sle15sp6 37.1 pip install failed

- Related ticket: https://jira.suse.com/browse/TEAM-8787
- Needles: NA
- Verification run:
    https://openqa.suse.de/tests/13101296#dependencies (test cases passed)

### CODE REVIEW

File: tests/sles4sap/hana_install.pm [hunk 1]
```diff
@@ -258,8 +264,9 @@ sub run {
     my @hdblcm_args = qw(--autostart=n --shell=/bin/sh --workergroup=default --system_usage=custom --batch
       --hostname=$(hostname) --db_mode=multiple_containers --db_isolation=low --restrict_max_mem=n
       --userid=1001 --groupid=79 --use_master_password=n --skip_hostagent_calls=n --system_usage=production
-      --components=server);
+    );
     push @hdblcm_args,
+      "--components=server,client",
```
Comment (mpagot) [inline tests/sles4sap/hana_install.pm:269]: The `client` part is needed by this other test module https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/monitoring_services.pm#L52

I propose to add a comment in the code to explain it

**Review by mpagot — APPROVED** [review]
LGTM

File: tests/sles4sap/hana_install.pm (hunk 1 above)
Comment (lilyeyes) [reply tests/sles4sap/hana_install.pm:271]: > The `client` part is needed by this other test module https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/monitoring_services.pm#L52
> 
> I propose to add a comment in the code to explain it

Done.

**Comment by alvarocarvajald** [comment]
Thanks a lot for this @lilyeyes . I was also taking a look at this on #18338 as I'm trying to fix the WMP test for HANA 2 SPS07 (see: https://gitlab.suse.de/lpalovsky/wmp_basic_tests/-/merge_requests/13). I've triggered some tests using your branch and SPS07:

- sle-15-SP6-Online-x86_64-Build45.1-SAPHanaSR_ScaleUp_PerfOpt_WMP_node01@64bit-sap -> http://openqaworker15.qa.suse.cz/tests/273285
- sle-15-SP6-Online-x86_64-Build45.1-SAPHanaSR_ScaleUp_PerfOpt_WMP_node02@64bit-sap -> http://openqaworker15.qa.suse.cz/tests/273283

**Review by alvarocarvajald — APPROVED** [review]
LGTM.

**Pull request merged**